pkgs/development/libraries/avro-c++/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, cmake, boost155, python
+}:
+
+stdenv.mkDerivation {
+  name = "avro-c++-1.7.5";
```

Comment:
To make updating a bit simpler, I recommend adding a `version` attribute and using it in the `name` & the `url` attrs.